### DISCUSSION TITLE
How can I remove the border of the notebook

### DISCUSSION THREAD

**Discussion by zSynctic** [rdbende/Azure-ttk-theme#25] [Question]
I want to tell you that you really did a good job in this theme it's so good.

I created a notebook which has tabs but then there is a border in each tab is there a way to remove and this is a screenshot of what I'm talking about:
![Screenshot 2022-02-06 143437](https://user-images.githubusercontent.com/71632495/152676860-7c13ca59-f2a1-437a-aed9-3c42bac3941a.png)

**Comment by rdbende** [comment]
Currently there it's not possible, but once I have free time for it I'll make it. 🙂

**Reply by zSynctic** [reply]
Oh ok thanks
